champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/passives/SmokeBomb.java

```diff
@@ -104,9 +104,11 @@ public double getDuration(int level) {
     public void toggle(Player player, int level) {
         // Effects
         championsManager.getEffects().addEffect(player, EffectType.INVISIBILITY, (long) (getDuration(level) * 1000L));
+        player.addPotionEffect(new PotionEffect(PotionEffectType.INVISIBILITY, (int)(getDuration(level) * 20), 0));
```

Comment:
do it in EffectListener when someone receives Invisibility. PotionEffect also takes an additional argument which hides particles